korman/exporter/mesh.py

```diff
@@ -387,6 +499,26 @@ def _export_geometry(self, bo, mesh, materials, geospans, mat2span_LUT):
         color = self._find_vtx_color_layer(mesh.tessface_vertex_colors, autocolor=not lm.bake_lightmap, manual=True)
         alpha = self._find_vtx_alpha_layer(mesh.tessface_vertex_colors)
 
+        # And retrieve the vertex groups that are deformed by an armature.
+        armatures = self._objects_armatures.get(bo.name)
+        export_deform = armatures is not None
+        if export_deform:
+            # We will need to remap IDs of each bone per armature usage. This is annoying, especially since we support multiple armatures...
+            i = 1
+            all_bone_names = {}
+            for armature in armatures:
+                for bone in armature.data.bones:
+                    all_bone_names[bone.name] = i
+                    i += 1
+            # This will map the group ID (used by Blender vertices) to the bone index exported to Plasma.
+            # Theoretically supports multiple armatures, except if the two armatures have the same bone names (because that would be REALLY asking for a lot here).
+            # If the bone is not found, we'll just map to the null bone.
+            group_id_to_bone_id = [all_bone_names.get(group.name, 0) for group in bo.vertex_groups]
+            # We will also need to know which bones deform the most vertices per material, for max/pen bones.
+            for gd in geodata.values():
+                gd.total_weight_by_bones = { j: 0.0 for j in range(i) }
```

Comment:
```suggestion
            bone_count = len(all_bone_names)
            for gd in geodata.values():
                gd.total_weight_by_bones = { i: 0.0 for i in range(bone_count) }
```

Could this be a list instead of a dict?